src/GitHub.App/ViewModels/RepositoryCloneViewModel.cs

```diff
@@ -80,9 +80,24 @@ public class RepositoryCloneViewModel : BaseViewModel, IRepositoryCloneViewModel
                 loading => loading.Value || repositories.UnfilteredCount > 0 && !LoadingFailed)
                 .ToProperty(this, x => x.FilterTextIsEnabled);
 
-            this.WhenAny(x => x.IsLoading, x => x.LoadingFailed,
-                (loading, failed) => !loading.Value && !failed.Value && repositories.UnfilteredCount == 0)
-                .Subscribe(x => NoRepositoriesFound = x);
+            this.WhenAny(
+                x => x.repositories.UnfilteredCount,
```

Comment:
The failing test appears to be a bug in `TrackingCollection` which duplicates items when the collection is initiated via `Listen` instead of in the ctor, and a `Filter` is set.